StreamChatClient_v3/WebSocketClient/WebSocketPingController_Tests.swift

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatClient_v3
+import XCTest
+
+class WebSocketPingController_Tests: XCTestCase {
+    var time: VirtualTime!
+    var pingController: WebSocketPingController!
+    var pingCount = 0
+    var reconnectCount = 0
+    
+    override func setUp() {
+        super.setUp()
+        time = VirtualTime()
+        VirtualTimeTimer.time = time
+        
+        pingCount = 0
+        reconnectCount = 0
+        
+        pingController = .init(timerType: VirtualTimeTimer.self,
+                               timerQueue: .main,
+                               ping: { [weak self] in self?.pingCount += 1 },
+                               forceReconnect: { [weak self] in self?.reconnectCount += 1 })
+    }
+    
+    func test_ping() throws {
```

Comment:
I think we need more tests. For example, you should test:
- that pings are sent periodically
- that pings are sent only when the connection is active
etc...